### PR TITLE
Document pull-req #6424 in release-notes

### PR DESCRIPTION
**Pull request by petertodd**

### CODE REVIEW

File: doc/release-notes.md [hunk 1]
```diff
@@ -105,6 +105,16 @@ In this version, it is only enforced for peers that send protocol versions
 removed. It is recommended to update SPV clients to check for the `NODE_BLOOM`
 service bit for nodes that report versions newer than 70011.
 
+Any sequence of pushdatas in OP_RETURN outputs now allowed
+----------------------------------------------------------
+
+Previously OP_RETURN outputs with a payload were only relayed and mined if they
+had a single pushdata. This restriction has been lifted to allow any
+combination of data pushes and numeric constant opcodes. (OP_1 to OP_16) The
```
Comment (paveljanik) [inline doc/release-notes.md]: The dot after parens?

Comment (petertodd) [inline doc/release-notes.md]: Hmm? What specifically do you think I should change?

Comment (paveljanik) [inline doc/release-notes.md]:
```
-constant opcodes. (OP_1 to OP_16)
+constant opcodes (OP_1 to OP_16).
```
ie. move the dot at the end.

Comment (petertodd) [inline doc/release-notes.md]: Ah! Sorry. Fixed.

File: doc/release-notes.md [hunk 2]
```diff
@@ -105,6 +105,16 @@ In this version, it is only enforced for peers that send protocol versions
...
 service bit for nodes that report versions newer than 70011.
 
+Any sequence of pushdatas in OP_RETURN outputs now allowed
+----------------------------------------------------------
+
+Previously OP_RETURN outputs with a payload were only relayed and mined if they
+had a single pushdata. This restriction has been lifted to allow any
+combination of data pushes and numeric constant opcodes (OP_1 to OP_16). The
+limit on OP_RETURN output size is now applied to the entire serialized
+scriptPubKey, 83 bytes by default. (the previous 80 byte default plus three
```
Comment (paveljanik) [inline doc/release-notes.md]: The same here. Sorry, I haven't seen it at first somehow.

**Comment by jonasschnelli** [comment]
ACK.

But I really would like to see git commit title/description understandable from a non github perspective. Keeping the mindset decentralized even at the place where the source code is managed is probably a good thing.

As @laanwj once mentioned, the issue/PR number within a commit title/description is okay. But still it should be understandable from a non GitHub perspective.

**Comment by luke-jr** [comment]
+1 on a readable commit message.

**Comment by petertodd** [comment]
@jonasschnelli Sure, tell me what you want it to be and I'll make it that.

**Comment by paveljanik** [comment]
What about: Mention now allowed sequence of pushdatas in OP_RETURN outputs in release notes.

**Comment by petertodd** [comment]
@paveljanik done

**Pull request merged**